components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -57,15 +56,29 @@ export default function Preferences() {
                 const updatedUser = await getGitpodService().server.getLoggedInUser();
                 setUser(updatedUser);
 
-                toast("Your default workspace timeout was updated.");
+                let toastMessage = <>"Your default workspace timeout was updated."</>;
+                if (billingMode.data?.mode === "usage-based") {
+                    if (!billingMode.data.paid) {
+                        toastMessage = (
+                            <>
+                                {toastMessage} Note that this will only affect workspaces in paid organizations. Go to{" "}
+                                <Link to="/billing" className="gp-link">
+                                    billing
+                                </Link>{" "}
+                                to upgrade your organization.
```

Comment:
**suggestion**: Copy rephrase:

> //BEFORE
> "Your default workspace timeout was updated." Note that this will only affect workspaces in paid organizations. Go to [billing]() to upgrade your organization.

> // AFTER
> Default workspace timeout has been updated. Changes are applicable only to workspaces within paid organizations.

Reply:
Done.